packages/webview_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.0
```

Comment:
It's not a big deal.. but how do you start a new version?

Reply:
Oh, I'm considering giving `webview_flutter_lwe` a new version instead. because webview_flutter_lwe will be newly published on pub.dev. What do you think about this?